seed/challenges/02-javascript-algorithms-and-data-structures/basic-javascript.json

```diff
@@ -4182,14 +4182,14 @@
     },
     {
       "id": "56533eb9ac21ba0edf2244c7",
-      "title": "Accessing Objects Properties with the Dot Operator",
+      "title": "Accessing Objects' Properties with the Dot Operator",
```

Comment:
I change my mind to something else. Here I think it would be best to be `Accessing Object Properties with the Dot Operator`, with no "s" on "object".